packages/discovery-provider/plugins/pedalboard/apps/verified-notifications/src/tracks.ts

```diff
@@ -0,0 +1,87 @@
+import { App } from "@pedalboard/basekit";
+import { Tracks } from "@pedalboard/storage";
+import { discoveryDb } from ".";
+import { sendSlackMsg } from "./slack";
+
+const TRACKS_SLACK_CHANNEL = process.env.TRACKS_SLACK_CHANNEL
+
+const onNewTrackRow = async (trackRow: Tracks) => {
+    const { track_id, updated_at, created_at } = trackRow
+    const isUpload =
+        updated_at === created_at &&
+        updated_at !== undefined &&
+        created_at !== undefined
+    if (!isUpload) return
+
+    const trackId = track_id
+    const results = await discoveryDb('tracks')
+        .innerJoin('users', 'tracks.owner_id', '=', 'users.user_id')
+        .innerJoin('track_routes', 'tracks.track_id', '=', 'track_routes.track_id')
+        .select(
+            'tracks.title',
+            'tracks.mood',
+            'tracks.genre',
+            'tracks.release_date',
+            'tracks.is_stream_gated',
+            'tracks.is_download_gated',
+            'tracks.owner_id',
+            'users.user_id',
+            'users.handle',
+            'users.name',
+            'tracks.track_id',
+            'users.is_verified',
+            'track_routes.slug',
+            'tracks.is_unlisted'
+        )
+        .where('tracks.track_id', '=', trackId)
+        .where('users.is_verified', '=', true)
+        .whereNull('tracks.stem_of')
+        .where('tracks.is_unlisted', '=', false)
+        .first()
+        .catch(console.error)
+
+    if (results === undefined) return
+
+    const { title, genre, mood, is_stream_gated, is_download_gated, handle, slug, release_date, name } =
+        results
+
+    const header = `:audius-spin: New upload from *${name}* 🔥`
+    const data = {
+        Title: title,
+        Genre: genre,
+        Mood: mood,
+        'Stream Gated': is_stream_gated,
+        'Download Gated': is_download_gated,
```

Comment:
nice!

Reply:
don't think this was me lol think Saliou added this a long time ago

Reply:
oh i don't see this in the channel. maybe we didn't we deploy?

Reply:
oh maybe we didn't, we'll get it with this!